app/controllers/concerns/bot_personal_access_token_actions.rb

```diff
@@ -54,9 +54,18 @@ def create # rubocop:disable Metrics/MethodLength
     end
   end
 
-  def revoke
+  def new_revoke
```

Comment:
Same type of comment as above. This should be named something like `revoke_confirmation` as this brings up the confirmation modal when we click on `Revoke` for a bot account token.

Reply:
Changed in 44216f0f3d059fa28a6e3abbd5f78ff3419ceaf8